images/openshift-ci/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# This Dockerfile is used by openshift CI
+# It builds an image containing snc code and nss-wrapper for remote deployments, as well as the google cloud-sdk for nested GCE environments.
+FROM scratch AS builder
+WORKDIR /code-ready/snc
+COPY . .
+
+FROM centos:7
+COPY --from=builder /code-ready/snc /opt/snc
+COPY --from=builder /code-ready/snc/images/openshift-ci/mock-nss.sh /bin/mock-nss.sh
+COPY --from=builder /code-ready/snc/images/openshift-ci/google-cloud-sdk.repo /etc/yum.repos.d/google-cloud-sdk.repo
+
+RUN yum update -y && \
+    yum install --setopt=tsflags=nodocs -y \
+    gettext \
+    google-cloud-sdk \
+    nss_wrapper \
+    openssh-clients && \
+    yum clean all && rm -rf /var/cache/yum/*
```

Comment:
Have you looked how hard it would be to get a base image for this? 2nd time I see this code duplicated, and I think the first time I saw it, it was already copied and pasted from another place..

Reply:
I also would like to see a consistent image got this between CRC and SNC

Reply:
To get a base image we need to maintain it, and one of the conversation which I had with Alex they really want to have base image as minimal as posible so in this case if centos:7 have new tag then our stuff auto run on new image but if we create ourself a base image name it `foo` then if there is any update on `centos:7` which will be base for `foo` first need to be updated so  I am OK with duplication as long as it is not maintenance burden.